pandas/tseries/resample.py

```diff
@@ -35,6 +35,7 @@ class TimeGrouper(Grouper):
     Use begin, end, nperiods to generate intervals that cannot be derived
     directly from the associated object
```

Comment:
if you wouldn't mind removing ``tseries/resample.py`` from this patch as I mostly rewritten this file in #11841 (I will PEP that itself).